pom.xml

```diff
@@ -23,7 +23,7 @@
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
     <jenkins.baseline>2.504</jenkins.baseline>
-    <jenkins.version>${jenkins.baseline}.1</jenkins.version>
+    <jenkins.version>2.519-rc37126.b_edb_579e2cf0</jenkins.version>
```

Comment:
@Vlatombe the upstream change was merged and available in [2.523](https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.523) (with  2.526 [scheduled](https://groups.google.com/g/jenkinsci-dev/c/zlZQTMyNQ8E) to be the next LTS).